src/lib/rest/rest.cpp

```diff
@@ -861,14 +863,12 @@ std::string defaultServicePath(const char* url, const char* method)
    */
   if (strcasestr(url, "contextAvailabilitySubscriptions") != NULL)      return DEFAULT_SERVICE_PATH_RECURSIVE;
   if (strcasestr(url, "contextSubscriptions") != NULL)                  return DEFAULT_SERVICE_PATH_RECURSIVE;
-  if (strcasecmp(method, "POST") == 0)                                  return DEFAULT_SERVICE_PATH;
-  if (strcasecmp(method, "PUT") == 0)                                   return DEFAULT_SERVICE_PATH;
-  if (strcasecmp(method, "DELETE") == 0)                                return DEFAULT_SERVICE_PATH;
-  if (strcasecmp(method, "GET") == 0)                                   return DEFAULT_SERVICE_PATH_RECURSIVE;
-  if (strcasecmp(method, "PATCH") == 0)                                 return DEFAULT_SERVICE_PATH;
 
-  std::string details = std::string("cannot find default service path for: (") + method + " " + url + ") - BAD VERB?";
```

Comment:
Actual change has been to remove just this line (i.e. all the rest shown and diff is about align changes and changes in the preamble comment)?